maturity-level-two/src/Startup.cs

```diff
@@ -28,6 +31,17 @@ public void ConfigureServices(IServiceCollection services)
             services.ConfigureOpenApiGeneration();
             services.ConfigureRouting();
             services.ConfigureInvalidStateHandling();
+            services.AddMvc(options =>
```

Comment:
Why source code has been added in a PR about the guideline? That's not what we agreed.